packages/base-styles/_mixins.scss

```diff
@@ -479,9 +479,3 @@
 		@include admin-scheme(#dd823b);
 	}
 }
-
-// It is important to include these styles in all built stylesheets
-// This allows to CSS variables post CSS plugin to generate fallbacks.
-:root {
-	@include admin-scheme(#007cba);
```

Comment:
interesting!

If we remove this from here, we need to move it to all the `edit-*` stylesheets + `list-reusable-blocks`.
Alternatively, we can define the fallback variables in the config of the PostCSS CSS variables plugin.

Reply:
I believe it's already there, through the mixin:

<img width="439" alt="Screenshot 2020-08-06 at 15 14 08" src="https://user-images.githubusercontent.com/1204802/89536006-85ce3880-d7f7-11ea-9a7c-fc747ad9407e.png">

And that mixin is something like

```
@mixin wordpress-admin-schemes() {
	body.admin-color-light {
		@include admin-scheme(#0085ba);
	}

 	...
}
```

I'm happy to test the failure condition of removing this if there's an easy way to do it?

Reply:
the fallback can be tested on IE11 or you can just check that anytime the variable is used on the produced CSS, there's always a default value applied.

Thinking again, moving this to `edit-*` won't solve the issue of fallbacks because the fallback won't be applied to stylesheets like "components"... So the only way is to actually define the fallback in the plugin's config.

Reply:
How do we best move forward with this one?